contracts/ROSCA.sol

```diff
@@ -0,0 +1,224 @@
+pragma solidity ^0.4.4;
+
+/**
+*  Things still missing/needs fixing:
+*   in withdraw() , add fee related logic
+*   checkingContribution, check whether or not a member as contributed as least contributionSize at the end of Round
+*/
+contract ROSCA {
+  address constant WETRUST = 0x0;
```

Comment:
one is put there by you, during our meeting at starbucks , its for triggering escape hatch
should we take it out completely?